api_tests/nodes/views/test_node_draft_registration_detail.py

```diff
@@ -737,257 +735,3 @@ def test_reviewer_cannot_delete_draft_registration(
             auth=user.auth, expect_errors=True)
         assert res.status_code == 403
         assert res.json['errors'][0]['detail'] == exceptions.PermissionDenied.default_detail
-
-
-@pytest.mark.django_db
-class TestDraftPreregChallengeRegistrationMetadataValidation(
```

Comment:
This is also testing legacy schema schema blocks behavior I believe there are no nested schemas anymore.